jazz_test-lambda/components/error-handler.js

```diff
@@ -0,0 +1,43 @@
+/**

```

Comment:
header missing here and other files as well

Reply:
done